.github/ISSUE_TEMPLATE/sample_app_proposal.yaml

```
@@ -0,0 +1,52 @@
+name: 🚀 Sample App Proposal
```

below list is our current issues menu.

- 🐞 Bug report
- Custom issue template
- Feature request
- 🚀 Sample App Proposal

The first letters each words are in uppercase, but I think it would be better to unify them to lowercase.
like below :)

```suggestion
name: 🚀 Sample App Proposal
```

I think it will be last modification request :)
After this, let me approve it! thanks for your quick response!

---

Then is it better to change title and labels field, too?

---

I think it will be better to change only name. not labels field.
labels name `Sample Proposal` is correct :)

---

title is also fine :) uppercase looks good to me!

---

Okay! I will apply your feedback!